### PR TITLE
Add route dashboard to grafana instance

### LINKED ISSUE #1737: Per-route historical metrics
Linkerd has metrics at the service level. These are valuable for understanding the general health of a service. Unfortunately, it is usually a specific route that is having problems. While it is possible to run `top` and get some metrics today, these are gathered in real time and not stored in prometheus.

It should be possible to see per-route metrics alongside the existing per-service metrics (success rate, latency, throughput). These can improve the time to fixing issues and provide visibility into what is really happening. 

## User Stories

- As a service owner, I would like to see per-route metrics in my dashboards so that I can quickly see any endpoints that are operating outside my SLO.
- As a service owner, I would like to see a list of all the routes in my service and sort that list by success rate, so that I can quickly see what is currently failing.
- As a service owner, I would like to have per-route metrics aggregated by URL parameters such as user id, so that I can quickly see what code path is being taken.
- As a service owner, I would like to persist per-route metrics so that I can use them to debug historical issues.

### PR DESCRIPTION
**Pull request by lundbird**
Fixes #1737

This adds a grafana dashboard to the linkerd grafana instance that gives visibility into the per route historical metrics of a service for latency, requests per second, and success rate. The dashboard has namespace, app, destination, route, and status_code as variables to segment the panels.

Signed-off-by: alex lundberg <alex.lundberg@commonbond.co>

### CODE REVIEW

**Comment by grampelberg** [comment]
This is really awesome, I've been wanting a dashboard for per-route metrics forever.

**Comment by lundbird** [comment]
Its possible your looking at the wrong timespace, as variables are evaluated by timespan, but its possible there is a misconfig somewhere. I'll spin up a test instance and see what is going on

**Comment by lundbird** [comment]
@alpeb Thanks for the feedback. I implemented your changes, in addition to some other changes to make this dashboard consistent with the othes, where it made sense to. 

As far as testing goes, I am installing booksapp/linkerd on a kind cluster, building and deploying a new grafana image, and viewing the dashboard. The issue you had above is because I had my timespan set to a specific span instead of -5 mins like the other dashboards.

File: grafana/dashboards/routes.json [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 1,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
```
[the remark this answers is not in the document]
Comment (lundbird) [reply grafana/dashboards/routes.json]: Should swap this out with the deployment variable?

**Comment by lundbird** [comment]
@alpeb I rethought how this should be implemented. The latest commit changes the dashboard to be a clone of the authority-dashboard, but now with a route variable added. This should be much more consistent with the other dashboards.

**Comment by lundbird** [comment]
I would be happy to link this in to the linkerd dashboard in a follow-up mr. 

Apps sometimes share the same routes like for healthchecks and this would aggregate calls between applications which is misleading, but I don't see another way to make this linkable from the linkerd dashboard. 

I've implemented your changes in the latest commit.

**Pull request merged**